### DISCUSSION TITLE
How to set default value for SelectActions?

### DISCUSSION THREAD

**Discussion by mahfuz-rahman007** [filamentphp/filament#14860] [Help]
### Package

Actions

### Package Version

3.2

### How can we help you?

I want to add a language dropdown to a create form. However, this dropdown is located outside the form. When the user selects a different language option, I need to modify the visibility of the form accordingly. I attempted to use SelectAction, but I can't set a default value because that method doesn't exist. Can anyone help me achieve this?

**Comment by leandrocfe** [comment]
Could you share the code that you are using?
